### PR TITLE
Upgrade Cython for Python 3.11

### PR DESCRIPTION
**Pull request by cclauss**
A minimal subset of #78

An on-demand GitHub Action:
```
name: pip_install_cchardet
on: [pull_request, push, workflow_dispatch]
jobs:
  pip_install_cchardet:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/setup-python@v4
        with:
          python-version: 3.11
      - run: python3.11 -m pip install --upgrade cython>=0.29.30
      - run: python3.11 -m pip install cchardet  # Will succeed only if cython>=0.29.30
```

### CODE REVIEW

**Comment by jameshilliard** [comment]
@PyYoshi Think we could get a new release with this updated as python 3.11 is now released?

**Review by SimplicityGuy — APPROVED** [review]

**Comment by SimplicityGuy** [comment]
Without this fix, this is preventing many downstream packages from being operational. @PyYoshi, please take a look and create a new release to include support for Python 3.11.

File: Pipfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ verify_ssl = true
 chardet = "==3.0.4"
 nose = "==1.3.7"
 tox = "==3.20.1"
-Cython = "==0.29.21"
+Cython = "==0.29.30"
```
Comment (SimplicityGuy) [inline Pipfile:10]: Latest version is `0.29.32`. Consider upgrading to that version.

**Comment by cclauss** [comment]
Closing in favor of #78 which really should be merged.

**Pull request closed without merging**